### PR TITLE
 [MIG] pos_payment_entries_globalization: Migration to 11.0

### PR DESCRIPTION
**Pull request by chienandalu**
This module allows to globalize payment entries created by the POS.

In some cases, all banking transactions are received in a single bank statement
line. With this module, it's possible to reconcile this one line with the
payment globalization line.

cc @Tecnativa

### CODE REVIEW

**Comment by rafaelbn** [comment]
@chienandalu great! Please traslate to spanish the labels. See:



cc @carlosdauden

**Review by rafaelbn — APPROVED** [review]
Tested functionaly :+1:

File: pos_payment_entries_globalization/README.rst [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+~~~~~~~
+
+* ACSONE SA/NV
+
+Contributors
+~~~~~~~~~~~~
+
+* Adrien Peiffer <adrien.peiffer@acsone.eu>
+* Meyomesse Gilles <meyomesse.gilles@gmail.com>
+* David Vidal <david.vidal@tecnativa.com>
```
Comment (rafaelbn) [inline pos_payment_entries_globalization/README.rst]: Have I contributed to this module?

**Review by rafaelbn — APPROVED** [review]

File: pos_payment_entries_globalization/README.rst (hunk 1 above)
Comment (chienandalu) [reply pos_payment_entries_globalization/README.rst]: Sure :smile:

**Review by carlosdauden — APPROVED** [review]

**Comment by pedrobaeza** [comment]
My only comment is about the name: it contains plurals and I'm not sure it reflects the proper meaning. Do you mean a global payment?

**Comment by rafaelbn** [comment]
> This module allows to globalize payment entries created by the POS.
> 
> In some cases, all banking transactions are received in a single bank statement line.
> With this module, it's possible to reconcile this one line with the payment globalization line.

Ref: https://github.com/OCA/pos/pull/70#issue-53703388
https://github.com/OCA/pos/tree/8.0/pos_payment_entries_globalization

**Review by yajo — APPROVED** [review]
Code OK.

**Comment by pedrobaeza** [comment]
I don't like too much the name in plural, but letting that question for now. For v12, it should be considered.

**Pull request merged**